sdk/template/Azure.Template/CHANGELOG.md

```diff
@@ -1,8 +1,26 @@
 # Release History
 
-## 1.0.3-beta.2 (Unreleased)
+## 1.0.3-beta.8 (Unreleased)
 
 
+## 1.0.3-beta.7 (2020-09-04)
```

Comment:
It isn't necessary to fill in all these older versions for the template project, but they also don't hurt.